kaggle/api/kaggle_api_extended.py

```diff
@@ -323,8 +323,8 @@ class KaggleApi(KaggleApi):
     ]
 
     # Datasets valid types
-    valid_dataset_file_types = ['all', 'csv', 'sqlite', 'json', 'bigQuery']
-    valid_dataset_license_names = ['all', 'cc', 'gpl', 'odb', 'other']
+    valid_dataset_file_types = ['unspecified', 'csv', 'sqlite', 'json', 'bigQuery']

```

Comment:
Alternatively, we could leave these as `'all'` to keep the values uniform across Comps/Notebooks. Then on 1152-1153, do a simple conversion from `'all'` to `'unspecified'`.